docs/run/quantum-serverless.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Run workloads with quantum serverless
+description: Run workloads using quantum serverless
+---
+
+# Run workloads with quantum serverless
+
+Premium users can run their workloads remotely on classical compute made available through IBM Quantum Platform.
+
+For more information, see the [quantum serverless docs](https://qiskit-extensions.github.io/quantum-serverless/).
+
+<Admonition type="note">
+    This is an experimental feature, subject to change.
+</Admonition>
+
+## Build a workflow
+
+Here is an example of computing the quasi-probability distribution of a quantum circuit using the Qiskit Runtime Estimator primitive. This Python script should be saved in your working directory. (Warning! All contents of the working directory will be shipped to the cluster for execution.)
+
+```python
+# source_files/my_qiskit_pattern.py
+
+from qiskit.circuit.random import random_circuit
+from qiskit.quantum_info import SparsePauliOp
+from qiskit_ibm_runtime import QiskitRuntimeService, Estimator
+from quantum_serverless import save_result
+
+service = QiskitRuntimeService()
+
+# Run on a simulator
+backend = service.get_backend("ibmq_qasm_simulator")
+# Use the next line if you want to run on a system
+# backend = service.least_busy(simulator=False)
+
+circuit = random_circuit(2, 2, seed=1234)
+observable = SparsePauliOp("IY")
+
+estimator = Estimator(backend)
+job = estimator.run(circuit, observable)
+result = job.result()
+
+# save results of program execution
+save_result(result)
+```
+
+## Deploy workflow on Quantum Serverless
+
+After creating a workflow, authenticate to the `IBMServerlessProvider` with your [IBM Quantum token](account-management), and upload the script.
+
+```python
+# Authenticate to the IBM serverless provider
+from quantum_serverless import IBMServerlessProvider
+serverless = IBMServerlessProvider("YOUR_IBM_QUANTUM_TOKEN")
+
+# Deploy the workflow
+from quantum_serverless import QiskitPattern
+serverless.upload(
+    QiskitPattern(
+        title="My-Qiskit-Pattern",
+        entrypoint="my_qiskit_pattern.py",
+        working_dir="./source_files/"
```

Comment:
just checking if this is the right path for this example

Reply:
yes, should be... we're assuming the script is what's noted in the comment in `source_files/my_qiskit_pattern.py` in https://github.com/Qiskit/documentation/blob/0131c29dbffe4d6a662104ba93f8a13ea1df1390/docs/run/quantum-serverless.mdx#L21